src/screen/DashboardScreen.tsx

```diff
@@ -35,8 +35,7 @@ function DashboardScreen({navigation}: PropTypes) {
 
   React.useLayoutEffect(() => {
     navigation.setOptions({
-      headerLeft: DashboardHeaderLeft,
-      headerRight: () => <View />,
```

Comment:
I think without header right it looks a bit weird on android. can you check that?